### ISSUE TITLE
Fix code owners for use with performance

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/fenix#15273]
We wrote a patch to use it to catch possible performance regressions https://github.com/mozilla-mobile/fenix/pull/14245 but code owners isn't working – we should investigate and fix it. I checked with liuche and it wasn't intentionally disabled.

**Comment by mcomella** [comment]
The root cause is probably a syntax error. via [this blog](http://www.benjaminoakes.com/git/2018/08/10/Testing-changes-to-GitHub-CODEOWNERS/):

> I’ve discovered what I believe to be a bug in CODEOWNERS. In particular, a single line failing to parse causes the entire file to stop identifying codeowners. This is in contrast to .gitignore, in which a single line failing to parse simply causes that line to be skipped. This is very surprising behavior and may have silently caused other users of CODEOWNERS to stop having their rules applied, as it did for us.

**Comment by mcomella** [comment]
It seems kind of problematic that anyone changing this file might break our important performance protections – maybe we should 1) run a codeowners validation script or 2) add ourselves as code owners to code owners.

**Comment by mcomella** [comment]
Using the technique described [in this blog post](http://www.benjaminoakes.com/git/2018/08/10/Testing-changes-to-GitHub-CODEOWNERS/), I validated that all of the paths we've entered into our code owners are valid. If there is actually a syntax error, this probably means it's within the names rather than the paths.

> It seems kind of problematic that anyone changing this file might break our important performance protections – maybe we should 1) run a codeowners validation script or 2) add ourselves as code owners to code owners.

I'm concerned about doing 1) because we'd be running unknown third party code. Since this file doesn't change often, we probably should do 2 but it is a little weird for the perf team to own the code owners file. This problem might not happen again so we could also ignore it until it happens again at which point we try to fix it more comprehensively.
